enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -132,11 +137,31 @@ export function getEmbeddingComponentOverrides(
       defaultProps: {
         withinPortal: true,
         portalProps: {
-          target: `#${EMBEDDING_SDK_ROOT_ELEMENT_ID}`,
+          target: `#${EMBEDDING_SDK_PORTAL_CONTAINER_ELEMENT_ID}`,
         },
 
         ...(theme?.popover?.zIndex && { zIndex: theme.popover.zIndex }),
       },
     },
+    ModalRoot: {
+      defaultProps: {
+        withinPortal: true,
+        target: `#${EMBEDDING_SDK_PORTAL_CONTAINER_ELEMENT_ID}`,
+      }, //  satisfies Partial<ModalRootProps>,
```

Comment:
I think the `satisfies` PR have been merged, so you can use that now 👍🏻

Reply:
We can, but mantine docs says they don't typecheck these by default because of performance, I'm 80% sure that the issue is typing all of the components and we're fine with just adding the `satisfies`s we need, but I'd still keep them commented just to play it safe for now